packaging/curated-cloud-installer.spec

```diff
@@ -0,0 +1,137 @@
+#
+# spec file for package curated-cloud-installer
+#
+# Copyright (c) 2022 SUSE LLC
+#
+# All modifications and additions to the file contributed by third parties
+# remain the property of their copyright owners, unless otherwise agreed
+# upon. The license for this file, and modifications and additions to the
+# file, is the same license as for the pristine package itself (unless the
+# license for the pristine package is not an Open Source License, in which
+# case the license is the MIT License). An "Open Source License" is a
+# license that conforms to the Open Source Definition (Version 1.9)
+# published by the Open Source Initiative.
+
+# Please submit bugfixes or comments via https://bugs.opensuse.org/
+#
+%define         app_dir      %{_datadir}/curated-cloud-installer
+%define         lib_dir      %{_libdir}/curated-cloud-installer
+%define         data_dir     %{_sharedstatedir}/curated-cloud-installer
+
+%define         ruby_version ruby3.0
+%define         k8s_version  kubernetes1.18
+
+Name:           curated-cloud-installer
+Version:        0.0.1
+Release:        0
+Summary:        Curated Cloud Installation of Rancher on AWS EKS
+License:        GPL-3.0
+Group:          Productivity/Networking/Web/Frontends
+URL:            http://www.github.com/suse-enceladus/curated-cloud-installer
+Source0:        %{name}-%{version}.tar.bz2
+Source1:        curated-cloud-installer.rpmlintrc
+BuildRequires:  %{ruby_version}
+BuildRequires:  %{ruby_version}-devel
+BuildRequires:  chrpath
+BuildRequires:  gcc
+BuildRequires:  sqlite3-devel
+Requires:       %{ruby_version}
+Requires:       %{k8s_version}-client
+Requires:       helm
+Requires:       nginx
+# Does not build for i586, s390, aarch64 and is not supported on those architectures
+ExcludeArch:    %{ix86} s390 aarch64
+
+%description
+Simple, usable web application for deploying complex applications to the cloud;
+wrapping cloud native SDK/CLIs
+
+%prep
+
+%setup
+sed -i '1 s|/usr/bin/env\ ruby|/usr/bin/ruby.%{ruby_version}|' bin/*
+
+%build
+
+# Drop 'BUNDLED WITH' line from Gemfile.lock. It causes trouble when the Gemfile.lock
+# was created with a different major version than the distribution's bundler.
+sed -i '/BUNDLED WITH/{N;d;}' Gemfile.lock
+
+bundle.%{ruby_version} config build.nio4r --with-cflags='%{optflags} -Wno-return-type'
+bundle.%{ruby_version} config set deployment 'true'
+bundle.%{ruby_version} config set without 'test development'
+bundle.%{ruby_version} install --local
+
+%install
+mkdir -p %{buildroot}%{app_dir}
+mkdir -p %{buildroot}%{lib_dir}
+mkdir -p %{buildroot}%{data_dir}
+
+mv vendor %{buildroot}%{lib_dir}
+mv db %{buildroot}%{data_dir}
+mv tmp %{buildroot}%{data_dir}
+mv log %{buildroot}%{data_dir}
```

Comment:
I think it's better to use the `install` command here than `mv` since you will also be able to set the desired permissions.

Also, you normally never delete files from the build tree (which is what `mv` does unlike `cp` and `install`).

Reply:
It's a convenience for mapping the layout from what Rails provides, to what SLES expects.